File: Modul 5/C/PRAK505.c
#include <stdio.h>
void Biodata(int tahunLahir, char Namaku[20], char Asal[15]){
    int tahun_sekarang = 2020;
// Lengkapi Function ini
    printf("Perkenalkan Nama Saya : %s\n", Namaku);
    printf("Umur Saya : %d\n", tahun_sekarang - tahunLahir);
    printf("Saya adalah angkatan : %d\n", tahun_sekarang);
    printf("Asal saya dari : %s\n", Asal);
}
int main() {
int tahunLahir;
 char Namaku[20], Asal[15];
 scanf(" %d",&tahunLahir);
 scanf(" %[^\n]%*c",&Namaku);
 scanf(" %[^\n]%*c",&Asal);
Biodata(tahunLahir, Namaku, Asal);
return 0;
}

File: Modul 5/Python/PRAK505.py
def Biodata(tahunLahir, Namaku, Asal):
    tahun_sekarang = 2020
    print(f"Perkenalkan Nama Saya : {Namaku}")
    print(f"Umur Saya : {tahun_sekarang - tahunLahir}")
    print(f"Saya adalah angkatan : {tahun_sekarang}")
    print(f"Asal saya dari : {Asal}")

tahunLahir = int(input())
Namaku = input()
Asal = input()
Biodata(tahunLahir, Namaku, Asal)